src/styles/_catalog.scss

```
@@ -1,22 +1,39 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-gap: 48px;
   max-width: 944px;
   margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
```

Here is the reason. Create a variable for the card width and use it 
`$card-width: 200px;`
```suggestion
    grid-template-columns: repeat(4, $card-width);
```